instrumentation/opentelemetry-instrumentation-pika/src/opentelemetry/instrumentation/pika/utils.py

```diff
@@ -40,16 +40,15 @@ def decorated_callback(
         body: bytes,
     ) -> Any:
         if not properties:
-            properties = BasicProperties()
-        if properties.headers is None:
-            properties.headers = {}
+            properties = BasicProperties(headers={})
         ctx = propagate.extract(properties.headers, getter=_pika_getter)
         if not ctx:
             ctx = context.get_current()
         span = _get_span(
             tracer,
             channel,
             properties,
+            span_kind=SpanKind.CLIENT,
```

Comment:
shoudl this be producer/consumer instead of client/server?

Reply:
Yeah, You create a single connection and declare one or many channels, over which you declare one or many callbacks. So it is more of a producer/consumer than client/server.